backend/drive/drive.go

```diff
@@ -1693,6 +1693,11 @@ func (f *Fs) listRRunner(ctx context.Context, wg *sync.WaitGroup, in chan listRE
 	var paths []string
 	var grouping int32
 
+	usingQueryFilter := false
```

Comment:
I don't think this test is working - when I tried it it always seemed to set `usingQueryFilter = true`

I was trying the example here:

https://github.com/rclone/rclone/issues/4289#issuecomment-636016383

Which you can probably try too since the folder is public.